### ISSUE TITLE
About reproducing the performance of the paper.

### ISSUE THREAD

**Issue by jang-1996** [PRBonn/contrastive_association#3]
Hello,

I trained the model using the code provided on your GitHub, and on the validation set I obtained the following performance. 
```
PQ4D:   0.6055207438587965
AQ_mean:        0.5775458010525248
iou_mean:       0.6348507262542885
things_iou:     0.6198925613538145
stuff_iou:      0.6457293916364514
```

Then, I ran the test set using the model checkpoint you shared, and submitted the results to the [SemanticKITTI 4D Panoptic Segmentation competition](https://codalab.lisn.upsaclay.fr/competitions/7097). The reported LSTQ score was 62.11.

Could you please provide some advice on reproducing the reported performance?
